go/teams/audit.go

```diff
@@ -360,8 +410,14 @@ func (a *Auditor) doPreProbes(m libkb.MetaContext, history *keybase1.AuditHistor
 	for _, tuple := range probeTuples {
 		m.Debug("preProbe: checking probe at merkle %d", tuple.merkle)
 		if tuple.team != keybase1.Seqno(0) || !tuple.linkID.IsNil() {
-			return 0, NewAuditError("merkle root at %v should have been nil for %v; got %s/%d",
-				tuple.merkle, history.ID, tuple.linkID, tuple.team)
+			return 0, NewAuditError("merkle root should not have had a leaf for team %v: got %s/%d at merkle seqno %v",
+				history.ID, tuple.linkID, tuple.team, tuple.merkle)
+		}
+		if tuple.hiddenResp.RespType == libkb.MerkleHiddenResponseTypeNONE && tuple.merkle > firstRootWithHidden && auditMode != keybase1.AuditMode_STANDARD_NO_HIDDEN {
+			return 0, NewAuditError("did not get a hidden response but one was expected (at main seqno %v)", tuple.merkle)
+		}
+		if tuple.hiddenResp.RespType != libkb.MerkleHiddenResponseTypeABSENCEPROOF && tuple.hiddenResp.RespType != libkb.MerkleHiddenResponseTypeFLAGOFF {
```

Comment:
are we actually using these yet?

Reply:
we are using absence proof (they can either be absence proofs or inclusion proofs for empty leaves). 
However, the whole pr is meant to be merged after we start putting the hidden chain root in the main tree in prod.